tensorflow/stream_executor/cuda/cuda_blas.h

```diff
@@ -84,21 +84,33 @@ class CUDABlas : public blas::BlasSupport {
   template <typename FuncT, typename... Args>
   bool DoBlasInternalImpl(FuncT cublas_func, Stream *stream,
                           bool pointer_mode_host, bool err_on_failure,
-                          Args... args);
+                          bool use_tensor_op_math, Args... args);
 
   // Convenience functions that call DoBlasInternalImpl with different values
-  // for err_on_failure.
+  // for err_on_failure and use_tensor_ops.
   template <typename FuncT, typename... Args>
   bool DoBlasInternal(FuncT cublas_func, Stream *stream, bool pointer_mode_host,
                       Args... args) {
     return DoBlasInternalImpl(cublas_func, stream, pointer_mode_host,
-                              /*err_on_failure=*/true, args...);
+                              /*err_on_failure=*/true, /*use_tensor_ops=*/false,
+                              args...);
+  }
+  template <typename FuncT, typename... Args>
+  bool DoBlasInternalWithTensorOps(FuncT cublas_func, Stream *stream,
+                                   bool pointer_mode_host, bool use_tensor_ops,
+                                   Args... args) {
+    return DoBlasInternalImpl(cublas_func, stream, pointer_mode_host,
+                              /*err_on_failure=*/true, /*use_tensor_ops=*/false,
```

Comment:
The name DoBlasInternalWithTensorOps() suggests that use_tensor_ops should be set to true. Also, if we use this function, I think the argument use_tensor_ops is redundant.

Reply:
Please also change the interfaces of  DoBlasGemmWithProfiling() and DoBlasGemmWithAlgorithm() for fp16. Thank you!

Reply:
Tensor op usage is encoded in the algorithm, so no modification of DoBlasGemmWithAlgorithm is needed.
DoBlasGemmWithProfiling wraps DoBlasGemm and also should not require further change as I understand the code.

Reply:
Yes you're right, both functions call DoBlasGemm() so nothing should be done in those two functions. Thanks for the correction.